josh-0000/listingPage
# Listing description page

Added a link to the listings that leads to the listing description page.
To Test:
1. Pull code
2. Ensure that pressing the listing takes you to listing description page
